lib/build-metadata.js

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
+"use strict";
+
+/**
+ * @typedef {Object} metadata
+ * @property {metadataProperty} *
```

Comment:
It also doesn't really make sense unless the variable is injected in. For example I might define `shelfLib~Reference` well thats sort of ok (I guess, even though its not a namespace) because that is what I called the parameter.

If I say this is `cwsShelfUpload~metadataProperty` where did `cwsShelfUpload` come from? Its not the name of my module, because thats `cws-shelf-upload`.